CLI_QUICKSTART.markdown

```diff
@@ -344,4 +344,19 @@ with a pkill command:
 $ pkill vc-server
 $ pkill vc-pas
 ```
+##  IP addresses and ports used by various Veracruz components
+
+|    IP      | Port |
+|------------|------|
+| 127.0.0.1  | 6000 |
```

Comment:
We are more interested in how those IPs are used in Veracruz. You may find that information in the comment lines where the IP appears in the source code.

Reply:
> We are more interested in how those IPs are used in Veracruz. You may find that information in the comment lines where the IP appears in the source code.

Ok will do that too.